src/server/frontend_wayland/wl_surface.cpp

```diff
@@ -670,7 +670,10 @@ void mf::WlSurface::set_buffer_scale(int32_t scale)
 
 void mir::frontend::WlSurface::offset(int32_t x, int32_t y)
 {
-    pending.offset = geom::Displacement{x, y};
```

Comment:
This was there for 4 years… why is it only a problem now?

What do we do to ensure this stays unbroke?

Reply:
It has not been four years since May 6th

Reply:
Still, anything we can easily to to ensure this remains unbroken?